components/core/src/EncodedVariableInterpreter.hpp

```diff
@@ -116,10 +114,9 @@ class EncodedVariableInterpreter {
 
 private:
     // Variables
-    // The beginning of the range used for encoding variable dictionary IDs
-    static constexpr encoded_variable_t m_var_dict_id_range_begin = 1LL << 62;
-    // The end (exclusive) of the range used for encoding variable dictionary IDs
-    static constexpr encoded_variable_t m_var_dict_id_range_end = (1ULL << 63) - 1;
+    // The maximum value for encoding variable dictionary IDs
+    static constexpr encoded_variable_t m_var_dict_id_max =
```

Comment:
Instead of using a method to get the max of `variable_dictionary_id_t`, can we use `cVariableDictionaryIdMax` instead? I like that you're using `std::numeric_limits` here, so maybe change the declaration of `cVariableDictionaryIdMax` in `Defs.h`.